lms/djangoapps/verify_student/views.py

```diff
@@ -195,6 +197,19 @@ class PayAndVerifyView(View):
     VERIFICATION_DEADLINE = "verification"
     UPGRADE_DEADLINE = "upgrade"
 
+    def _get_user_active_status(self, user):
```

Comment:
Since this is intended to be an internal/private function inside of the `verify_student' application itself, I think we're okay here.